project/src/components/property-screen/reviews-list.tsx

```diff
@@ -0,0 +1,26 @@
+import Review from './review';
+import { Reviews } from '../../types/offers-type';
+
+type ReviewListType = {
+  reviews: Reviews;
+};
+
+function ReviewsList(props: ReviewListType): JSX.Element {
```

Comment:
Здесь у тебя два множественных числа, надо убрать либо `s` либо List

```suggestion
function ReviewList(props: ReviewListType): JSX.Element {
```